packages/razzle/config/createConfig.js

```diff
@@ -473,13 +469,7 @@ module.exports = (
     } else {
       // Specify production entry point (/client/index.js)
       config.entry = {
-        client: [
-          // We ship a few polyfills by default but only include them if React is being placed in
-          // the default path. If you are doing some vendor bundling, you'll need to require the razzle/polyfills
-          // on your own.
-          !dotenv.raw.REACT_BUNDLE_PATH && require.resolve('./polyfills'),
-          paths.appClientIndexJs,
-        ].filter(Boolean),
+        client: [paths.appClientIndexJs],
```

Comment:
```suggestion
        client: paths.appClientIndexJs, 
```

This will enable treeshaking in production. Right @thelarkinn?